src/frontend/app/features/services/services-wall/services-wall.component.html

```diff
@@ -0,0 +1,22 @@
+<app-page-header [endpointIds$]="cfIds$">
+  <h1>Services</h1>
+  <div class="page-header-right">
+    <button mat-icon-button [routerLink]="'/services/new/'">
```

Comment:
Probably best leaving this at for the moment. Got some exceptions in the console + ended in at `https://localhost:4200/services/new/summary` with a blank page

Reply:
I'll leave this in, since its addressed in https://github.com/cloudfoundry-incubator/stratos/pull/2163